src/backend/distributed/planner/merge_planner.c

```diff
@@ -283,70 +253,6 @@ ErrorIfDistTablesNotColocated(Query *parse, List *distTablesList,
 }
 
 
-/*
```

Comment:
All the code is moved and simplified into the caller of this routine `ErrorIfMergeHasUnsupportedTables`